### PR TITLE
Task as/j135 select cover podcast image functionality

### PR DESCRIPTION
**Pull request by qwiz1**
2021-08-13 16_17_23-Jabber
2021-08-13 16_18_56-Jabber
Edit podcast page
image

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by JuliaDzyuba — DISMISSED** [review]

**Review by artem-totality — DISMISSED** [review]

File: frontend/src/common/enums/ui/image-preview-control-style.enum.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+enum ImagePreviewStyle {
```
Comment (what1s1ove) [inline frontend/src/common/enums/ui/image-preview-control-style.enum.ts]: remove it

File: frontend/src/components/common/image-preview-control/image-preview-control.tsx [hunk 2]
```diff
@@ -12,12 +12,17 @@ import { ErrorMessage } from '@hookform/error-message';
...
+import { ImagePreviewStyle } from 'common/enums/ui/image-preview-control-style.enum';
 
 type Props = {
   name: Path<FieldValues>;
   control: Control;
   errors: FieldValues;
   imageUrl?: string;
+  label?: string;
+  className?: string;
+  imagePreviewStyle?: ImagePreviewStyle;
```
Comment (what1s1ove) [inline frontend/src/components/common/image-preview-control/image-preview-control.tsx]:
```suggestion
```

Comment (what1s1ove) [reply frontend/src/components/common/image-preview-control/image-preview-control.tsx]: you className to give styles for this block

File: frontend/src/components/common/image-preview-control/image-preview-control.tsx [hunk 3]
```diff
@@ -40,15 +53,28 @@ const ImagePreviewControl: React.FC<Props> = ({ name, control, errors, imageUrl
...
+    styles[`style${imagePreviewStyle}`],
+  );
+
   return (
     <>
-      <label className={styles.inputWrapper}>
+      <label className={allowedClassesInputWrapper}>
+        <div className={styles.cornerRight}>
+          <span className={styles.label}>{label}</span>
+        </div>
```
Comment (what1s1ove) [inline frontend/src/components/common/image-preview-control/image-preview-control.tsx]:
```suggestion
        {label && 
        <span className={styles.cornerRight}>
          {label}
        </span>}
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/configurate-podcast/components/configurate-podcast-form/styles.module.scss [hunk 4]
```diff
@@ -22,4 +22,8 @@
       width: 100%;
     }
   }
+
+  .imagePreview{
+    max-width: 358px;
```
Comment (what1s1ove) [inline frontend/src/components/configurate-podcast/components/configurate-podcast-form/styles.module.scss:27]:
```suggestion
    max-width: 358px;
    margin: 0 auto;
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**